packages/eslint-config-airbnb-base/rules/imports.js

```diff
@@ -82,7 +82,7 @@ module.exports = {
         '**/rollup.config.*.js', // rollup config
         '**/gulpfile.js', // gulp config
         '**/gulpfile.*.js', // gulp config
-        '**/Gruntfile', // grunt config
+        '**/Gruntfile{,.js,.coffee}', // grunt config
```

Comment:
I'm fine supporting the `.js` extension, but nobody should be using coffeescript. Can you remove `.coffee`?